include/arch/arm/arch/32/mode/object/structures.bf

```diff
@@ -199,9 +199,8 @@ block VMFault {
 
 #ifdef CONFIG_ARM_HYPERVISOR_SUPPORT
 block VGICMaintenance {
-    field idx        6
-    field idxValid   1
-    padding         25
+    field eisr      64
```

Comment:
Fields in the bitfield specs must be lower than the word width of the machine, so this field here is giving you compile errors.

Reply:
Thanks for the help, I still have a compilation issue:
```
CMakeFiles/kernel.elf.dir/src/arch/arm/32/traps.S.obj CMakeFiles/kernel.elf.dir/src/arch/arm/32/hyp_traps.S.obj CMakeFiles/kernel.elf.dir/kernel_all.c.obj -o kernel.elf   && :
  /usr/lib/gcc-cross/arm-linux-gnueabi/10/../../../../arm-linux-gnueabi/bin/ld: CMakeFiles/kernel.elf.dir/kernel_all.c.obj: in function `VGICMaintenance.part.0':
  kernel_all.c:(.text+0x9[65](https://github.com/seL4/seL4/actions/runs/5114599391/jobs/9195008723?pr=1046#step:4:67)c): undefined reference to `__ctzdi2'
```

I will check if I can solve this soon